packages/firestore/src/protos/google/firestore/v1/firestore.proto

```diff
@@ -522,6 +545,62 @@ message RunQueryResponse {
   int32 skipped_results = 4;
 }
 
+// The request for [Firestore.RunAggregationQuery][google.firestore.v1.Firestore.RunAggregationQuery].
+message RunAggregationQueryRequest {
+  // Required. The parent resource name. In the format:
+  // `projects/{project_id}/databases/{database_id}/documents` or
+  // `projects/{project_id}/databases/{database_id}/documents/{document_path}`.
+  // For example:
+  // `projects/my-project/databases/my-database/documents` or
+  // `projects/my-project/databases/my-database/documents/chatrooms/my-chatroom`
+  string parent = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
In andriod-sdk proto file, this line is stated as `string parent = 1; ` .
Other required field in web-sdk firestore.proto always trailed by `[(google.api.field_behavior) = REQUIRED]`.   So I added the same for this line.
Is this correct?

Reply:
I think you are right to append the "REQUIRED" thing. It probably doesn't really matter. The proto in google3 does indeed have the REQUIRED clause: http://google3/google/firestore/v1beta1/firestore.proto;l=703;rcl=440401402. So it's probably good to have it there.